public/locales/en.yml

```diff
@@ -3,7 +3,8 @@ en:
     request_list:
       aeon_button_label: My HOLLIS Special Request Account
       request_form_subtitle: SUBMIT REQUEST
-      empty_list_message: To add items to this list use "Add to My List" when viewing materials you want to request. Return to this screen to submit requests. <br><br> <b>Need to manage existing requests?</b> Log in to <span class="hvd_hollis-login-link"><a href="https://aeontest.hul.harvard.edu/logon" target="harvard-library-requests">HOLLIS Special Request<i class="fa fa-external-link" aria-hidden="true"></i></a></span>.
+      empty_list_message: To add items to this list use "Add to My Request List" when viewing materials you want to request. Return to this screen to submit requests. <br><br>
+      #existing_list_message: <b>Need to manage existing requests?</b> Log in to <span class="hvd_hollis-login-link"><a href="https://aeontest.hul.harvard.edu/logon" target="harvard-library-requests">HOLLIS Special Request<i class="fa fa-external-link" aria-hidden="true"></i></a></span>.
```

Comment:
Suggest removing this line rather than committing a commented-out line, unless there's a pressing reason, and if there is, it should be documented with a comment.

Reply:
Thanks for catching that!